.github/workflows/tekton_bundle_push.yaml

```diff
@@ -17,21 +15,58 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: debianmaster/actions-k3s@master
-        id: k3s
-        with:
-          version: 'latest'
+      - run: |
+          mkdir -vp $HOME/{bin,.kube} || true
+          pushd $HOME/bin
+          curl -LO https://dl.k8s.io/release/v1.23.0/bin/linux/amd64/kubectl
+          popd
+          PATH="$HOME/bin:$PATH"
+          cat <<-EOF > $HOME/.kube/config
```

Comment:
You can use something even more minimal:
```
apiVersion: v1
kind: Config
clusters:
- cluster:
    server: _
  name: _
contexts:
- context:
    cluster: _
  name: _
current-context: _
```
Ideally [this PR](https://github.com/tektoncd/cli/pull/1557) will be merged though. Something for the future though.